mlpf/pyg_pipeline.py

```diff
@@ -50,7 +50,7 @@
 parser.add_argument("--test", action="store_true", default=None, help="tests the model")
 parser.add_argument("--num-epochs", type=int, default=None, help="number of training epochs")
 parser.add_argument("--patience", type=int, default=None, help="patience before early stopping")
-parser.add_argument("--lr", type=float, default=None, help="learning rate")
+parser.add_argument("--lr", type=float, default=0.001, help="learning rate")
```

Comment:
This needs to be `None` by default for the overriding of config parameters to work properly. Defaults can be specified in the config files instead of here. If you prefer to have defaults specified here, we would need to modify the `override_config()` function in `pyg_pipeline.py`.

Reply:
I was getting this error without it
```python
Traceback (most recent call last):
  File "/home/joosep/particleflow/mlpf/pyg_pipeline.py", line 435, in <module>
    main()
  File "/home/joosep/particleflow/mlpf/pyg_pipeline.py", line 431, in main
    device_agnostic_run(config, args, world_size, outdir)
  File "/home/joosep/particleflow/mlpf/pyg_pipeline.py", line 336, in device_agnostic_run
    run(rank, world_size, config, args, outdir, logfile)
  File "/home/joosep/particleflow/mlpf/pyg_pipeline.py", line 89, in run
    optimizer = torch.optim.AdamW(model.parameters(), lr=args.lr)
  File "/opt/conda/lib/python3.10/site-packages/torch/optim/adamw.py", line 28, in __init__
    if not 0.0 <= lr:
TypeError: '<=' not supported between instances of 'float' and 'NoneType'
```

Reply:
I addressed it here: https://github.com/jpata/particleflow/pull/257/commits/d816282a781919f778a4332134ca620df32c8893